students/etwum/lesson03/slicing.py

```diff
@@ -0,0 +1,147 @@
+
+def exchange_first_last(seq):
+    # exchanges the first/last items in a sequence then copies the new sequence into a new string, list or tuple
+
+    # performs this part of the if statement if the sequence is of type string
+    if type(seq) is str:
```

Comment:
`isinstance` check is preferred over `type` checks:
https://github.com/UWPCE-PythonCert-ClassRepos/Self_Paced-Online/pull/103/files